applications/asset_tracker_v2/src/watchdog/watchdog.c

```diff
@@ -91,15 +91,16 @@ static int watchdog_feed_enable(struct wdt_data_storage *data)
 		return err;
 	}
 
-	err = k_delayed_work_submit(&data->system_workqueue_work,
+	err = k_work_schedule(&data->system_workqueue_work,
 				    K_MSEC(WDT_FEED_WORKER_DELAY_MS));
-	if (err) {
+	if (err < 0) {
```

Comment:
Same as in the button handler, I believe this is no longer possible in normal scenarios

Reply:
fixed